### PR TITLE
Video store

### PR DESCRIPTION
**Pull request by janisrudis**

### CODE REVIEW

File: src/exercises/oop/Ts.videoStore/video-store.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+export class Video {
+  name: string;
+  rating: number;
+  user: string;
+  constructor(name: string, rating: number) {
```
Comment (sandris-) [inline src/exercises/oop/Ts.videoStore/video-store.ts]: where are ratings? :(

Comment (janisrudis) [reply src/exercises/oop/Ts.videoStore/video-store.ts]: Ts don't work. JS is completely done.
I can try to do that too, but as I hear from classmates, they did all homewors in JS. And they said that you accepted.

Comment (janisrudis) [reply src/exercises/oop/Ts.videoStore/video-store.ts]: Ts don't work. JS is completely done.
I can try to do that too, but as I hear from classmates, they did all homewors in JS. And they said that you accepted.

Comment (sandris-) [reply src/exercises/oop/Ts.videoStore/video-store.ts]: do you want to be best or acceptable?

File: src/exercises/oop/fuel.ts/car.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      this.fuelGauge.decrease(1);
+      this.counter = 0;
+    }
+  }
+}
+
+Odometer.minMileage = 0;
+Odometer.maxMileage = 999999;
+
+// const fuelGauge = new FuelGauge(10);
```
Comment (sandris-) [inline src/exercises/oop/fuel.ts/car.ts]: why do you need commented code in git?

File: src/exercises/oop/fuel.ts/car.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  amount: number;
+  static minAmount: number;
+  static maxAmount: number;
+  constructor(amount: number) {
+    this.amount = amount;
+  }
+
+  decrease(amount: number) {
+    if (this.amount - amount >= FuelGauge.minAmount) {
+      for (let i = 0; i < amount; i++) this.amount--;
```
Comment (sandris-) [inline src/exercises/oop/fuel.ts/car.ts]: for loop looks awful

File: src/exercises/oop/video-store/index.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    {
+      name: "userRating",
+      type: "input",
+      message: "Enter movie rating :"
+    }
+  ]);
+
+  store.addRating(movie, userRating);
+
+  // console.log("##Returned works?##Available to rent", store.printContent);
```
Comment (sandris-) [inline src/exercises/oop/video-store/index.js]: ?

File: src/exercises/oop/video-store/video-store.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  userRating(userRating) {
+    this.rating.push(userRating);
+    let convertToNumbers = this.rating.map(Number);
+    let total = convertToNumbers.reduce((acc, c) => acc + c, 0);
+    this.votes = convertToNumbers.length;
+    this.avgRating = total / this.votes;
+
+    // console.log(this.avgRating);
```
Comment (sandris-) [inline src/exercises/oop/video-store/video-store.js]: ?

File: src/exercises/oop/video-store/video-store.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+class VideoStore {
+  constructor() {
+    this.movies = [];
+  }
+
+  addMovie(name, rating) {
+    this.movies.push(new Video(name, rating));
+    // addRating(name, rating);
```
Comment (sandris-) [inline src/exercises/oop/video-store/video-store.js]: ?

**Pull request merged**